### DISCUSSION TITLE
Deploying fine-tuned HuggingFace models using FastAPI

### DISCUSSION THREAD

**Discussion by engageintellect** [vercel/community#4104] [Help]
### Summary

I'm trying to deploy a HuggingFace model using the transformers, fastAPI, and uvicorn packages. I keep getting the following error:

```
Error: A Serverless Function has exceeded the unzipped maximum size of 250 MB. : https://vercel.link/serverless-function-size
```

Is there a recommended way around this? Or should I just stick to running these types of models off of a linode server with uvicorn and nginx?

### Example

https://github.com/engageintellect/heyhey

### Steps to Reproduce

My code appears to work fine locally. Can reproduce my error by pushing the repo in the example link to vercel.

**Comment by Jfeng3** [comment]
had same issue.

**Comment by engageintellect** [comment]
> had same issue.

Did you you find a way to work around this that you would recommend?
